core/zoe/contracts/publicAuction.js

```diff
@@ -20,75 +20,111 @@ export const makeContract = harden((zoe, terms) => {
   const ITEM_INDEX = 0;
   const BID_INDEX = 1;
 
-  const publicAuction = harden({
-    startAuction: async escrowReceipt => {
-      const {
-        offerHandle,
-        offerRules: { payoutRules },
-      } = await zoe.burnEscrowReceipt(escrowReceipt);
+  const makeBidderSeat = inviteHandle =>
+    harden({
+      bid: () => {
+        // Check that the item is still up for auction
+        const { inactive } = zoe.getOfferStatuses(
+          harden([creatorInviteHandle]),
+        );
+        if (inactive.length > 0) {
+          return rejectOffer(
```

Comment:
Since `rejectOffer` itself only throws, it is arguably better that you should `throw rejectOffer(` rather than `return rejectOffer(`. For example, with `return` a possible type checker might try to check the return type of `rejectOffer` against the return type of `bid`.

Reply:
Good idea